GoogleMapsApi.Test/IntegrationTests/DistanceMatrixTests.cs

```diff
@@ -243,8 +240,8 @@ public void ShouldPassRawDataToOnRawResponseRecivied()
             try
             {
                 var result = GoogleMaps.DistanceMatrix.Query(request);
-                if (result.Status == DistanceMatrixStatusCodes.OVER_QUERY_LIMIT)
-                    Assert.Inconclusive("Cannot run test since you have exceeded your Google API query limit.");
+
+                AssertInconclusive.NotExceedQuota(result);
```

Comment:
I'm wondering if there's a way to write it once somewhere and run for all tests instead of the duplication of this line

Reply:
Not sure. Maybe some kind of http interceptor can help. However, I haven't dived so deep in the code :)